api/src/main/java/org/openmrs/api/impl/PatientServiceImpl.java

```diff
@@ -574,6 +577,13 @@ public void mergePatients(Patient preferred, Patient notPreferred) throws APIExc
 			log.debug("Merge operation cancelled: Cannot merge user" + preferred.getPatientId() + " to self");
 			throw new APIException("Patient.merge.cancelled", new Object[] { preferred.getPatientId() });
 		}
+		
+		 if (patientMergeActions != null) {
+	            for (PatientMergeAction patientMergeAction : patientMergeActions) {
+	                patientMergeAction.beforeMergingPatients(preferred, notPreferred);
+	            }
+	        }
```

Comment:
Can you fix these lines so they have the same indentation has the rest of the file?